homeassistant/components/lutron/config_flow.py

```diff
@@ -68,3 +79,36 @@ async def async_step_user(
             ),
             errors=errors,
         )
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: ConfigEntry,
+    ) -> OptionsFlowHandler:
+        """Get the options flow for this handler."""
+        return OptionsFlowHandler()
```

Comment:
Tests are missing.

Reply:
I [added a test for the OptionsFlowHandler](https://github.com/home-assistant/core/compare/dev...cameronr:home-assistant-core:lutron-default-brightness). Should I open another PR to submit it?

Reply:
Yes, please. Thanks!